build.sbt

```
@@ -11,12 +11,8 @@ crossScalaVersions := Seq("2.11.12", "2.12.10", scalaVersion.value)
 
 scalacOptions ++= Seq("-deprecation", "-feature")
 
-scalacOptions ++= {
-  if (scalaVersion.value startsWith "2.11")
-    Seq("-target:jvm-1.7")
-  else
-    Seq.empty
-}
+scalacOptions += "-target:jvm-1.8"
+javacOptions ++= Seq("-source", "8", "-target", "8")
```

As long as we support scala 2.11, we need to emit jvm7 for this version
```suggestion
scalacOptions ++= {
  if (scalaVersion.value startsWith "2.11")
    Seq("-target:jvm-1.7")
  else
    Seq("-target:jvm-1.8")
}
javacOptions ++= {
  if (scalaVersion.value startsWith "2.11")
    Seq("-source", "7", "-target", "7")
  else
    Seq("-source", "8", "-target", "8")
}
```